### DISCUSSION TITLE
Get user who made a change & subscribe to changes not made by user

### DISCUSSION THREAD

**Discussion by xRealNeon** [pocketbase/pocketbase#1361] [Q&A] [answered]
Hello, sorry when I am asking duplicate questions, but I could not find an answer.

1. Can I get the user who made a change if I subscribe to a collection?
2. Can I subscribe only to changes not made by the current user?

**Comment by ganigeorgiev** [comment, marked as answer]
> 1. Can I get the user who made a change if I subscribe to a collection?

No, unless you have a custom `relation` field in your collection like `updatedBy` or similar (note that depending on your collection API rules it is also possible that the change may be initialized by a non-auth/guest client).

> 2. Can I subscribe only to changes not made by the current user?

This is related to 1) and since we don't track by default who made the change, you can't filter by the current user. If you define a custom `updatedBy` field you can exclude client-side the record changes (eg. `e.record.updatedBy != pb.authStore.model.id`).
